build_deps.sh

```diff
@@ -25,6 +25,14 @@ llvm_branch=tags/RELEASE_700/final
 klee_repo=https://github.com/rsas/klee
 klee_branch=pure-bv-qf-llvm-7.0
 
+alivedir=third_party/alive2
+alive_builddir=$alivedir/build
+mkdir -p $alivedir $alive_builddir
+git clone git@github.com:manasij7479/alive2.git $alivedir/alive2
+(cd $alive_builddir && cmake ../alive2 -GNinja)
+ninja -C $alive_builddir
+
```

Comment:
what do you think about also building z3 so we can control the version?

Reply:
Let's wait until we use more Z3 calls directly in souper.